### ISSUE TITLE
[bitnami/redmine] Cannot start the console

### ISSUE THREAD

**Issue by ant-222** [bitnami/containers#71490]
### Name and Version

bitnami/redmine:4.1.1-5

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Launch `user_remine.bat` to enter the Redmine command line evnironment.
2. Execute `bundle exec rails c production` to lauch the Redmine Rails console.

### What is the expected behavior?

The Redmine Rails console starts.

### What do you see instead?

[These](https://pastebin.com/raw/nv2u9ya8) usage instructions for `rails new`.

### Additional information

This is not so much an assue as a help request. I cannot seem to find a way to start the Redmine Rails console for the management and administration of Redmine via the API, e.g. for user administration as described [here](https://www.redmine.org/boards/2/topics/8550). Am I doing something wrong, or is my setup somehow broken? Where can I seek help with this?

**Comment by ant-222** [comment]
Starting the Rails console in Bitnami Redmine:

0. Change to the *Redmine* installation root.
1. Run `use_redmine.bat`. Another console window will open.
2. Change to `apps\redmine\htdocs`.
3. Execute `SET RAILS_ENV=production`,  
   (specify the environment you need).
4. execute `bundle exec rails c`.

I wish this information were avaible in the official *Bitnami* documentaiton. This console works, but prints [this](https://pastebin.com/raw/RPDNXwUX) warning that it is obosolete. Is it correct behavior, or shall I really follow the instructions and Git in order to upgrade to the new *Rails* binstub?

**Comment by javsalgar** [comment]
Hi,

Are you using the Bitnami Redmine container or the Bitnami Redmine Virtual Machine?

**Comment by ant-222** [comment]
I was using the container.

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the feedback. About the warning, as containers are ephemeral, I wouldn't worry about it. About the console, would you like to submit a PR in the README.md file so the community can benefit from your instructions?

**Comment by ant-222** [comment]
I wrote:

> I was using the container.

I fear I misinformed you. I am using a program called *Bitnami Redmine Stack* that is installed locally on our Windows server. So, it is not a cloud solution, nor does it seem to be any kind of a container or virtual machine. Futhermore, I cannot find this product in the [Redmine section](https://bitnami.com/stack/redmine) of the *Bitnami* website. Is it discontinued in favour of cloud solutions and containers? Is there no longer a package isntallable as a normal component onto an existing *Windows* machine, without either virtualisation or containers?

**Comment by ant-222** [comment]
@javsalgar 

> About the warning, as containers are ephemeral, I wouldn't worry about it.

Well, I prefer doing things the clean way, and—as I wrote in the [previous message](https://github.com/bitnami/containers/issues/71490#issuecomment-2388845822)—I am not using a container, but a set of directly installed programs called *Bitnami Redmine Stack*.

> About the console, would you like to submit a PR in the README.md file so the community can benefit from your instructions?

Yes, I can do that, but I will need time to get around to it. Alternatively, I can post a snippet of *Markdown* text with the instructions.

**Comment by javsalgar** [comment]
Hi,

I'm confused, you mentioned before that you were using a container. By Bitnami Redmine Stack, do you mean a Virtual Machine? Could you provide the link to it?

**Comment by ant-222** [comment]
No, I mean normal applications, no VM and no docker. Is installed under
`C:\redmine-4.1.1-5\`
[READE.txt](https://pastebin.com/raw/s6XBkJyc) in that location is titled:
`Bitnami Redmine Stack 4.1.1-5`
and a bit below, it says:

>The Bitnami Redmine Stack is distributed as a binary executable installer. It can be downloaded from:
>
>https://bitnami.com/stacks/
>
>The downloaded file will be named something similar to:
>
> bitnami-redmine-4.1.1-5-linux-installer.run on Linux or
> bitnami-redmine-4.1.1-5-linux-x64-installer.run on Linux 64 bits or
> bitnami-redmine-4.1.1-5-windows-installer.exe on Windows or
> bitnami-redmine-4.1.1-5-osx-x86-installer.dmg on OS X x86.

So this is a normal installer of normal programs: no docker, no VM. Right?
